### PR TITLE
Store pickup get pickup location api test coverage#2264

### PR DESCRIPTION
**Pull request by vpodorozh**
### Description (#2264)
Cover Pickup Locations API with webapi tests.
See related PR #2214

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ishakhsuvarov** [comment]
@magento run Integration Tests build, Functional Tests build

File: app/code/Magento/InventoryInStorePickupApi/Test/Api/GetPickupLocations/CombinedTest.php [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+use Magento\InventoryInStorePickupApi\Api\Data\SearchRequest\DistanceFilterInterface;
+use Magento\TestFramework\Assert\AssertArrayContains;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class CombinedTest extends WebapiAbstract
+{
+    /**#@+
+     * Service constants
+     */
+    const RESOURCE_PATH = '/V1/inventory/in-store-pickup/pickup-locations';
```
Comment (swnsma) [inline app/code/Magento/InventoryInStorePickupApi/Test/Api/GetPickupLocations/CombinedTest.php]: Please make them private.
Please remove docs.

File: app/code/Magento/InventoryInStorePickupApi/Test/Api/GetPickupLocations/DistanceFilterOfflineTest.php [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+use Magento\InventoryInStorePickupApi\Api\Data\SearchRequest\DistanceFilterInterface;
+use Magento\TestFramework\Assert\AssertArrayContains;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class DistanceFilterOfflineTest extends WebapiAbstract
+{
+    /**#@+
+     * Service constants
+     */
+    const RESOURCE_PATH = '/V1/inventory/in-store-pickup/pickup-locations';
```
Comment (swnsma) [inline app/code/Magento/InventoryInStorePickupApi/Test/Api/GetPickupLocations/DistanceFilterOfflineTest.php]: Please make them private.
Please remove docs.

File: app/code/Magento/InventoryInStorePickupApi/Test/Api/GetPickupLocations/FilterSetTest.php [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+use Magento\InventoryInStorePickupApi\Api\Data\SearchRequest\DistanceFilterInterface;
+use Magento\TestFramework\Assert\AssertArrayContains;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class FilterSetTest extends WebapiAbstract
+{
+    /**#@+
+     * Service constants
+     */
+    const RESOURCE_PATH = '/V1/inventory/in-store-pickup/pickup-locations';
```
Comment (swnsma) [inline app/code/Magento/InventoryInStorePickupApi/Test/Api/GetPickupLocations/FilterSetTest.php]: Please make them private.
Please remove docs.

File: app/code/Magento/InventoryInStorePickupApi/Test/Api/GetPickupLocations/GeneralTest.php [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\TestFramework\Assert\AssertArrayContains;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class GeneralTest extends WebapiAbstract
+{
+    /**#@+
+     * Service constants
+     */
+    const RESOURCE_PATH = '/V1/inventory/in-store-pickup/pickup-locations';
```
Comment (swnsma) [inline app/code/Magento/InventoryInStorePickupApi/Test/Api/GetPickupLocations/GeneralTest.php]: Please make them private.
Please remove docs.

**Review by swnsma — CHANGES_REQUESTED** [review]
Maybe it makes sense to combine all tests in single test file with proper comments for test cases?

**Review by swnsma — CHANGES_REQUESTED** [review]
Thank you for your changes!
Almost done!

Could you please:
- resolve merge conflict
- add check not only by pickup location code, but compare attributes values (see \Magento\InventoryInStorePickupGraphQl\Test\Api\PickupLocationsPositiveCasesTest)

**Review by swnsma — APPROVED** [review]

**Pull request merged**